srml/timestamp/src/lib.rs

```diff
@@ -98,11 +98,16 @@ impl TimestampInherentData for InherentData {
 pub struct InherentDataProvider;
 
 #[cfg(feature = "std")]
+/// Implementation of ProvideInherentData
 impl ProvideInherentData for InherentDataProvider {
+	/// returns the identifier (storage key) for timestamp inherent data
 	fn inherent_identifier(&self) -> &'static InherentIdentifier {
 		&INHERENT_IDENTIFIER
 	}
 
+	/// provides the current system timestamp (in seconds from epoch) as timestamp inherent data
```

Comment:
You should not repeat the implementation of the function as documentation.

And please start with capital letters at the beginning of a documentation.